Tempo: Add link to Tempo Search with node service selected

joey-grafana opened the pull request:
**What this PR does / why we need it**:

This PR adds a link to Tempo Search with node service selected.

**Which issue(s) this PR fixes**:

Fixes https://github.com/grafana/grafana/issues/50075

**Special notes for your reviewer**:

![Screenshot 2022-05-29 at 14 00 31](https://user-images.githubusercontent.com/90795735/170869969-fcdf3d36-a943-4373-a4b9-5aeb8c5bf6d5.png)

joey-grafana committed:
Node to tempo search with service selected

joey-grafana committed:
Tests

connorlindsey reviewed (approved):
What do you think about the link text being something like 'View traces'? I think we use Tempo search a lot internally, but it might not be super clear what it means in the context of the link to users.

We should probably consider splitting the links into sections w/ headers to make them more scannable next time we add a link as well.

joey-grafana committed:
Update link text

joey-grafana committed:
Merge branch 'main' into joey/node-to-tempo-search

joey-grafana commented:
> What do you think about the link text being something like 'View traces'? I think we use Tempo search a lot internally, but it might not be super clear what it means in the context of the link to users.
> 
> We should probably consider splitting the links into sections w/ headers to make them more scannable next time we add a link as well.

Updated link text.

Yeah, splitting up the menu sounds good. I've added it to my todo list and will get to it soon.

joey-grafana committed:
Update test

joey-grafana committed:
Merge branch 'main' into joey/node-to-tempo-search
